### ISSUE TITLE
Need to be able to @Ignore fields

### ISSUE THREAD

**Issue by abhinavc** [vitusortner/floor#220]
Hi Vitus,
I have a database model for which I'm trying to define equality (overload `operator==`) using the `Equatable` package.

`Equatable` requires the following changes in one's model:
```dart
   class Model with EquatableMixin {
       .... 
       List<Object> get props => [ .... ] ; 
   }
```
where `props` is the list of data members that must be equal for two objects to be considered equal.

Problem is the `get props` line. `Floor`'s generator function seems to think that this is a database column and errors out with the following message

```
Error running FloorGenerator
Column type is not supported for List<Object>.
```

The solution would be to `@Ignore` the said line - as proposed in issues #12, #216 etc. Would it be possible, therefore, to get a quick patch release with this fix? The alternative would be for me to not use `Equatable` at all. 

Thanks
Abhinav.

**Comment by abhinavc** [comment]
Vitus,
For now, I have stopped using `Equatable` and overloaded `==` in whichever classes I needed to. So, no rush from my side. However, `@Ignore` would be useful ;-)

Thanks
Abhinav.

**Comment by vitusortner** [comment]
Closed by #248
